doc/getting-started.md

```diff
@@ -9,9 +9,9 @@ repository](https://github.com/alire-project/alire/releases).
 
 On Linux, `Alire` is simply provided in an archive.
 
-Once the archive is extracted you have to add `alr` in the environment `PATH`:
+Once the archive is extracted you have to add `alr` in the environment `PATH`in the .bashrc file of the user's home directory:
```

Comment:
I am not sure everyone will want to put that in `.bashrc`. I personally prefer to set the PATH every time I want to use Alire.

Reply:
Then I suppose it's a case of an _either ... or ..._ depending on the user's choice.
Personally I would see no danger in having this additional path entry and a lot of convenience in retaining it.

Reply:
This could be reworded as to indicate that it's an optional possibility and not a requirement for use.

Reply:
Edited as per mosteo suggestion.

Reply:
I guess this belongs in a **setup-environment.md** document.

Reply:
@jquorning 

Why not where it is right now, i.e. the "Getting Started" page ?
https://github.com/alire-project/alire/blob/master/doc/getting-started.md

Reply:
> Why not where it is right now, i.e. the "Getting Started" page ? 

I guess the environment is a big matrix of combinations of terminals, character sets, shells, configurations, etcetera not controlled by Alire. Especially on Linux. IMO Alire  should not try to adapt to all kinds of specialized use cases internally.